### PR TITLE
Android null result

### PR DESCRIPTION
**Pull request by MaximBelov**
### What do these changes do?

Standardise null result on android and ios

### Why are these changes necessary?

Should return a "null" result on Android  instead of "OK"
android – AirshipCordova kt  AirshipCordova app main  2024-04-19 16-04-55

### How did you verify these changes?

Run demo app

#### Verification Screenshots:

android – AirshipCordova kt  AirshipCordova app main  2024-04-19 16-07-01

iPhone
android – AirshipCordova kt  AirshipCordova app main  2024-04-19 16-07-01

### CODE REVIEW

File: cordova-airship/src/android/AirshipCordova.kt [hunk 1]
```diff
@@ -396,7 +396,7 @@ internal fun JsonSerializable.pluginResult(): PluginResult {
     val json = this.toJsonValue()
 
     return when {
-        json.isNull -> PluginResult(PluginResult.Status.OK)
+        json.isNull -> PluginResult(PluginResult.Status.OK, json.string)
```
Comment (rlepinski) [inline cordova-airship/src/android/AirshipCordova.kt:399]: instead of json.string should it just be `null` ?

Comment (MaximBelov) [reply cordova-airship/src/android/AirshipCordova.kt:399]: null is not supported by cordova 
Screenshot 2024-04-19 at 17 34 42

Comment (rlepinski) [reply cordova-airship/src/android/AirshipCordova.kt:399]: oh, this is literally passing "null"

Comment (rlepinski) [reply cordova-airship/src/android/AirshipCordova.kt:399]: Looks like we use to do an empty string for null - https://github.com/urbanairship/urbanairship-cordova/blob/14.11.0/urbanairship-cordova/src/android/UAirshipPlugin.java#L794

Comment (MaximBelov) [reply cordova-airship/src/android/AirshipCordova.kt:399]: Yes, an empty string is fine too

Comment (rlepinski) [reply cordova-airship/src/android/AirshipCordova.kt:399]: I am sure your change is good but let me double check the difference between iOS and Android

Comment (rlepinski) [reply cordova-airship/src/android/AirshipCordova.kt:399]: There is also https://documentation.opencms.org/javadoc/core/org/opencms/json/JSONObject.Null.html that we can try

Comment (rlepinski) [reply cordova-airship/src/android/AirshipCordova.kt]: It looks like it does support null - https://github.com/apache/cordova-android/blob/master/framework/src/org/apache/cordova/PluginResult.java#L42C46-L42C63

try `(String)null`

**Comment by rlepinski** [comment]
https://github.com/urbanairship/urbanairship-cordova/pull/428/files

**Comment by MaximBelov** [comment]
Fixed
https://github.com/urbanairship/urbanairship-cordova/pull/428/files#diff-9a808c9c8deee161f9c5b48f6bf9d6399a4d0604ca921a40ad7d59ffeb7f6004R385

**Comment by MaximBelov** [comment]
Fixed
https://github.com/urbanairship/urbanairship-cordova/pull/428/files#diff-9a808c9c8deee161f9c5b48f6bf9d6399a4d0604ca921a40ad7d59ffeb7f6004R385

**Pull request closed without merging**